src/SFA.DAS.PSRService.Application/ReportHandlers/CreateReportRequest.cs

```diff
@@ -10,17 +10,14 @@ public class CreateReportRequest : IRequest<Report>
             User user, 
             string employerId, 
             string period,
-            bool? isLocalAuthority)
+            bool isLocalAuthority)
         {
             if (string.IsNullOrWhiteSpace(employerId))
                 throw new ArgumentException("Value cannot be null or whitespace.", nameof(employerId));
 
             if (string.IsNullOrWhiteSpace(period))
                 throw new ArgumentException("Value cannot be null or whitespace.", nameof(period));
 
-            if (!isLocalAuthority.HasValue)
```

Comment:
why have we removed the null check? The only thing recommended here was to make the property non-null
because the parameter may be null, however the property is protected by the null check